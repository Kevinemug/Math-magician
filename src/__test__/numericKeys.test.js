import React from 'react';
import renderer from 'react-test-renderer';
import NumberKeys from '../components/numericKeys'

it('renders NumericKeys correctly', () => {
  const tree = renderer.create(<NumberKeys />)
  expect(tree).toMatchSnapshot()
});