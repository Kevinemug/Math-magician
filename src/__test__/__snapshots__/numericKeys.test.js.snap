// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NumericKeys correctly 1`] = `
<div
  className="numbers"
>
  <button
    className="btn btn-primary"
    data-testid="seven"
    type="button"
  >
    7
  </button>
  <button
    className="btn btn-primary"
    data-testid="eight"
    type="button"
  >
    8
  </button>
  <button
    className="btn btn-primary"
    data-testid="nine"
    type="button"
  >
    9
  </button>
  <button
    className="btn btn-primary"
    data-testid="four"
    type="button"
  >
    4
  </button>
  <button
    className="btn btn-primary"
    data-testid="five"
    type="button"
  >
    5
  </button>
  <button
    className="btn btn-primary"
    data-testid="six"
    type="button"
  >
    6
  </button>
  <button
    className="btn btn-primary"
    data-testid="one"
    type="button"
  >
    1
  </button>
  <button
    className="btn btn-primary"
    data-testid="two"
    type="button"
  >
    2
  </button>
  <button
    className="btn btn-primary"
    data-testid="three"
    type="button"
  >
    3
  </button>
  <button
    className="btn btn-primary col-span-2"
    data-testid="zero"
    type="button"
  >
    0
  </button>
  <button
    className="btn btn-primary"
    data-testid="full-stop"
    type="button"
  >
    .
  </button>
</div>
`;
